local-process-quickstart.sh

```diff
@@ -81,83 +65,67 @@ do
 done
 
 # Print helpFunction in case parameters are empty
-if [ ! -z "$CLEANUP" ]
-then
-   if [ -z "$RGNAME" ]
-   then
+if [ ! -z "$CLEANUP" ]; then
+   if [ -z "$RGNAME" ]; then
       echo "Please pass -g when calling -c"
       helpFunction
    else
       cleanupFunction
    fi
-elif [ -z "$RGNAME" ] || [ -z "$AKSNAME" ]
-then
+elif [ -z "$RGNAME" ] || [ -z "$AKSNAME" ]; then
    echo "Some or all of the parameters are empty";
    helpFunction
 fi
 
-if [ -z "$REPOROOT" ]
-then
-   echo "Defaulting Dev spaces repository root to current directory : ${PWD}"
-   REPOROOT=$PWD
+if [ -z "$REPOROOT" ]; then
+   echo "Defaulting Git repository root to current directory : $PWD"
+   REPOROOT="$PWD"
 fi
 
 installHelmFunction
 
 echo "Setting the Kube context"
 az aks get-credentials -g $RGNAME -n $AKSNAME
 
-echo "Applying role assignment"
-SUB=$(az account show --query id -o tsv)
-SPID=$(az aks show -n ${AKSNAME} -g ${RGNAME} --query servicePrincipalProfile.clientId -o tsv)
-if [[ "${SPID}" == "msi" ]]; then
-   # Managed identity cluster
-   SPID=$(az aks show -n ${AKSNAME} -g ${RGNAME} --query identity.principalId -o tsv)
-fi
-az role assignment create --assignee ${SPID} --scope "/subscriptions/${SUB}/resourceGroups/${RGNAME}" --role "Network Contributor"
-
-echo "Creating Public IP"
-AKSLOCATION=$(az aks show -n ${AKSNAME} -g ${RGNAME} --query location -o tsv)
-PUBLICIP=$(az network public-ip create --resource-group $RGNAME --name $PIPNAME --location $AKSLOCATION --sku Standard --allocation-method static --query publicIp.ipAddress -o tsv)
-echo "BikeSharing ingress Public IP: " $PUBLICIP
- 
-echo "Create namespace ${INGRESSNAME}"
-kubectl create namespace $INGRESSNAME
+echo "Create namespace dev"
+kubectl create namespace dev
 
 # Use Helm to deploy a traefik ingress controller
 echo "helm repo add && helm repo update"
-${HELMDIR}/helm repo add stable https://kubernetes-charts.storage.googleapis.com/
-${HELMDIR}/helm repo update
+$HELMDIR/helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$HELMDIR/helm repo update
 echo "helm install traefik ingress controller"
-${HELMDIR}/helm install $INGRESSNAME stable/traefik \
-   --namespace $INGRESSNAME \
+$HELMDIR/helm install $INGRESSNAME stable/traefik \
+   --namespace dev \
    --set kubernetes.ingressClass=traefik \
    --set fullnameOverride=$INGRESSNAME \
    --set rbac.enabled=true \
-   --set service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-resource-group"="$RGNAME" \
-   --set loadBalancerIP=$PUBLICIP \
    --set kubernetes.ingressEndpoint.useDefaultPublishedService=true \
    --version 1.85.0
- 
-NIPIOFQDN=${PUBLICIP}.nip.io
+
```

Comment:
Should we add a log statement before this loop like "Waiting for public IP assignment..."?